plugin/evm/vm.go

```diff
@@ -343,18 +366,51 @@ func (vm *VM) Initialize(
 	return nil
 }
 
-// Bootstrapping notifies this VM that the consensus engine is performing
-// bootstrapping
-func (vm *VM) Bootstrapping() error {
-	vm.bootstrapped = false
-	return nil
+func (vm *VM) initGossipHandling() {
+	if vm.chainConfig.SubnetEVMTimestamp != nil {
+		vm.gossiper = vm.newPushGossiper()
+		vm.Network.SetGossipHandler(NewGossipHandler(vm))
+	} else {
+		vm.gossiper = &noopGossiper{}
+		vm.Network.SetGossipHandler(message.NoopMempoolGossipHandler{})
+	}
```

Comment:
we shouldn't need this either since gossiping has existed for the full history of subnet-evm. Is it possible to remove `SubnetEVMTimestamp` or does it serve another purpose as well?

Reply:
We still treat SubnetEVMTimestamp as a hardfork because of legacy tests. I tried to remove it couple of times but some tests expect a pre-subnetevm genesis/fork.